starwars/src/components/StarWars.css

```diff
@@ -0,0 +1,67 @@
+.character-list {
```

Comment:
Using Percentages or rem/em instead of pixels can help make the component more responsive. Specifically, it increases the fluidity to match the screen size.